arcgis-ios-sdk-samples/Features/Feature layer selection/FeatureLayerSelectionViewController.swift

```diff
@@ -37,11 +37,13 @@ class FeatureLayerSelectionViewController: UIViewController {
     
     /// The feature layer created from a feature service.
     let featureLayer: AGSFeatureLayer = {
-        // Create feature table using a url.
-        let featureServiceURL = URL(string: "https://services1.arcgis.com/4yjifSiIG17X0gW4/arcgis/rest/services/GDP_per_capita_1960_2016/FeatureServer/0")!
```

Comment:
If you go to https://services1.arcgis.com/4yjifSiIG17X0gW4/ArcGIS/rest/services/GDP_per_capita_1960_2016/FeatureServer , you'll see its Service ItemId: 10d76a5b015647279b165f3a64c2524f